pkg/testutils/sqlutils/inject.go

```diff
@@ -24,7 +24,10 @@ import (
 
 // InjectDescriptors attempts to inject the provided descriptors into the
 // database.
-func InjectDescriptors(ctx context.Context, db *gosql.DB, input []*descpb.Descriptor) error {
+// If force is true, we can inject descriptors that are invalid.
+func InjectDescriptors(
+	ctx context.Context, db *gosql.DB, input []*descpb.Descriptor, force bool,
```

Comment:
nit: can you sprinkle `/* force */` comments everywhere you call `InjectDescriptors` with a boolean literal as argument, please?